src/admin/class-sign-in-with-google-admin.php

```diff
@@ -232,6 +232,14 @@ public function settings_api_init() {
 			'siwg_settings',
 			'siwg_section'
 		);
+	
+		add_settings_field(
+			'siwg_disable_login_page',
+			__( 'Disable WP login page and reditect users to Goolge Sign In', 'sign-in-with-google' ),
```

Comment:
```suggestion
			__( 'Skip the login page and redirect users to Google for authentication', 'sign-in-with-google' ),
```

Reply:
Sure, my PR's are not empirical -  means that you can absolutely modify the names or whatever, as you prefer, as long the concept/functionality will be achieved.